Step by step guide: use http-server instead of python

vicb opened the pull request:
- `http-server` is already used in the quick start guide,
- fontend devs are less likely to have Python installed

kwalrath commented on public/docs/js/latest/guide/setup.jade:
```diff
@@ -12,10 +12,17 @@
+    If you don't already have an HTTP server,
+    you can install one using <code>npm install -g http-server</code>.
+    (If that results in an access error, then you might need to use
+    <code><b>sudo</b> npm ...</code>).
```
). -> .)

kwalrath commented:
I happened to notice a small punctuation error. Someone more familiar with the JS dev process should probably review this.

vicb commented:
Fixed, thanks for noticing.

I don't think there is any need to review more, this can get in.

Quick question: what is `public/docs/js/latest/guide/setupAlt.jade` ? (The file I have modified is `public/docs/js/latest/guide/setup.jade` not sure what the Alt is)

kwalrath commented:
I believe the Alt file shows how to do examples in the new way, whatever that is. You might want to update that too.

vicb commented:
I have updated the alt file. Not sure How to test, I get a 404 when clicking on the menu entry (I did revert your last commit locally for the menu entry to appear).

kwalrath commented:
Oh, and... I accepted this because the alt page isn't required for this site, but did you restart the harp server? I think json file changes require restarting it.

vicb commented:
Yep I restarted `gulp watch-and-serve`, didn't help

kwalrath commented:
Sorry, I meant running harp compile again. I'm not sure whether watch-and-serve would be enough.

vicb commented:
Yep, also tried that. I actually tried:
- `$ harp compile`
- `$ harp server` 
- `$ gulp serve-and-watch`
